buf/registry/module/v1beta1/digest.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.registry.module.v1beta1;
+
+import "buf/validate/validate.proto";
+
+option go_package = "buf.build/gen/go/bufbuild/registry/protocolbuffers/go/buf/registry/module/v1beta1";
+
+// A digest of a Commit's content.
+//
+// A digest represents all content for a single Commit, including its .proto files, documentation
+// files, license files, and dependencies.
+//
+// TODO: should this be named something other than Digest, to make sure it is clear that this
+// is not related to the digest of a single File? This was originally named "CommitDigest" but
+// if we're trying to move away from Commits, this seems wrong.
```

Comment:
If we're moving away from Commits, what entity refers to a collection of files (proto, license, and documentation files) and dependencies to other collections of files? The Commit is the only thing that encapsulates this today. In the CLI we call this the Module, but Module refers to something else in the API.